R/f.R

```diff
@@ -6,8 +6,8 @@
 #' @export
 #' @examples
 #' new_f(quote(a))
-#' new_f(quote(a), quote(b))
-new_f <- function(rhs, lhs = NULL, env = parent.frame()) {
+#' new_f(quote(a), lhs = quote(b))
+new_f <- function(rhs, env = caller_env(), lhs = NULL) {
```

Comment:
I think they will be mostly used for testing NSE functions, e.g. you'll mostly find them in testthat sections of packages. `_f` is the suffix that correspond to the `f_` prefix so it seems consistent. I can also just revert the commit so we stop over-bikeshedding this :)